src/Reducer/filterReducer.jsx

```diff
@@ -9,28 +9,27 @@ const filterReducer = (state, action) => {
 
     case "SORT_BY":
       if (action.payload === "low-to-high") {
-        const updatedItems = [...state.items].sort((item1, item2) =>
-          parseInt(item1.price, 10) < parseInt(item2.price, 10) ? -1 : 1
+        const updatedItems = [...state.items].sort(
+          (item1, item2) => item1.price - item2.price
         );
         return {
           ...state,
           sortBy: "low-to-high",
           items: updatedItems,
         };
       } else if (action.payload === "high-to-low") {
```

Comment:
```high-to-low``` maybe you should put such strings in some object and use. There are high chances of missing a letter using hardcoded strings which in turn may break the app.